CHANGELOG.asciidoc

```diff
@@ -6,7 +6,7 @@ This file documents all notable changes to https://github.com/devonfw/ide[devonf
 
 WIP: Release with small but important bugfixes:
 
-* TODO: replace me
+* https://github.com/devonfw/ide/issues/903[#966]: Fixxed some issues with the npm detection.
```

Comment:
As this happend for the second time:
fixed is written with a single `x` :)
Also issue link and link-text do not match. should both be 966.
I will update DoD to explain that we always should put the exact title of the issue as description here via C&P.

Reply:
I updated this as I had to resolve merge conflict anyhow.

Reply:
Oh damn, sorry for the inconveniences. I will pay more attention to this in the future so this type of small mistakes dont happen again. And this is the first time I have ever heard, that it's called "fixed" thats good to know! 😄